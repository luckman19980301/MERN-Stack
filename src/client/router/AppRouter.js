import React from 'react';
import { HashRouter as Router } from 'react-router-dom';
import AppSwitch from './AppSwitch';

const AppRouter = (props) => (
    <Router { ...props }>
        <AppSwitch />
    </Router>
);

export default AppRouter;
