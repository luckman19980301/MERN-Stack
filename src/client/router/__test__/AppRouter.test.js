import React from 'react';
import { shallow } from 'enzyme';
import AppRouter from '../appRouter.jsx';

describe('AppRouter component', () => {
  const wrapper = shallow(<AppRouter />);

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
