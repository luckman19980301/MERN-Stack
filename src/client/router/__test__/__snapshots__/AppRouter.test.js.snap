// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppRouter component renders correctly 1`] = `ShallowWrapper {}`;
